src/AiidaDFTK.jl

```diff
@@ -160,34 +169,53 @@ function run_json(filename::AbstractString; extra_output_files=String[])
             JSON3.pretty(io, TimerOutputs.todict(DFTK.timer))
         end
     end
-    push!(all_output_files, timingfile)
 
-    (; output_files=all_output_files)
+    @info "$LOG_FINISHED_SUCCESSFULLY."
 end
 
 
 """
-Run a DFTK calculation from a json input file. The input file name is expected to be passed
-as the first argument when calling Julia (i.e. it should be available via `ARGS`. This
-function is expected to be called from queuing system jobscripts, for example:
+Run a DFTK calculation from a json input file. This function is expected
+to be called from queuing system jobscripts, for example:
 
 ```bash
-julia --project -e 'using AiidaDFTK; AiidaDFTK.run()' /path/to/input/file.json
+julia --project -e 'using AiidaDFTK; AiidaDFTK.run(inputfile="/path/to/input/file.json")'
 ```
+
+It automatically dumps a logfile `file.log` (i.e. basename of the input file
+with the log extension), which contains the log messages (i.e. @info, @warn, ...).
+Currently stdout and stderr are still printed.
+
+Required keyword arguments:
+- `inputfile`: The input file name.
+
+Optional keyword arguments:
+- `allowed_versions`: A range of supported AiidaDFTK versions, in the format supported by Pkg.
 """
-function run()
-    inputfile = only(ARGS)
+function run(; inputfile::AbstractString, allowed_versions::AbstractString="*")
```

Comment:
For sth like this I would naturally go with a positional argument ... but you may have your reasons.

Reply:
Oh I just changed it because
```py
'using AiidaDFTK; AiidaDFTK.run(inputfile="{}", allowed_versions="{}")'.format(...)
```
reads better to me than
```py
'using AiidaDFTK; AiidaDFTK.run("{}", allowed_versions="{}")'
```

Reply:
ok !